.github/ISSUE_TEMPLATE/1-roadmap-item.yml

```
@@ -0,0 +1,30 @@
+name: '📦 New Roadmap Item'
+labels: 'enhancement'
+description: >
+  This is a project which normally spans within a single across a few minor releases,
+  and that become major deliverables when shipped.
+  Potential examples are "Support for Java 22", "Problems HTML Summary Report", 
+  "Isolated Projects: Incubating Status" or "Declarative Gradle EAP 2".
+  Consider it "EPIC" in normal Agile terms.
+
+body:
+  - type: textarea
+    attributes:
+      label: Summary / Manifesto
+      description: > 
+        Describe what value this project brings to Gradle, and what areas/topics it should address.
```

❌ There should be an explicit pointer about user value. Consider adding an extra mandatory section for user-focused description - it might be different from the summary.

---

The target value is a whole purpose of the initial overview. I will extend so it is not "to Gradle" but to Gradle users or so

```suggestion
        Describe what value this project brings to Gradle end users or developers, and what areas/topics it will address.
```